From weiny2@llnl.gov  Wed Jul 21 11:52:36 2010
Date: Thu, 15 Jul 2010 11:34:44 -0700
From: Ira Weiny <weiny2@llnl.gov>
To: Patrick Mochel <mochel@osdl.org>,
	Mike Murphy <mamurph@cs.clemson.edu>
Cc: Randy Dunlap <rdunlap@xenotime.net>,
	Greg Kroah-Hartman <gregkh@suse.de>,
	Phil Carmody <ext-phil.2.carmody@nokia.com>,
	Lucian Adrian Grijincu <lgrijincu@ixiacom.com>,
	linux-kernel@vger.kernel.org, linux-doc@vger.kernel.org
Subject: sysfs: Fix one more signature discrepancy between sysfs implementation and docs.
Message-Id: <20100715113444.68bbe2dd.weiny2@llnl.gov>

Signed-off-by: Ira Weiny <weiny2@llnl.gov>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 Documentation/filesystems/sysfs.txt |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/Documentation/filesystems/sysfs.txt
+++ b/Documentation/filesystems/sysfs.txt
@@ -4,7 +4,7 @@ sysfs - _The_ filesystem for exporting k
 Patrick Mochel	<mochel@osdl.org>
 Mike Murphy <mamurph@cs.clemson.edu>
 
-Revised:    10 July 2010
+Revised:    15 July 2010
 Original:   10 January 2003
 
 
@@ -333,7 +333,7 @@ Structure:
 struct bus_attribute {
         struct attribute        attr;
         ssize_t (*show)(struct bus_type *, char * buf);
-        ssize_t (*store)(struct bus_type *, const char * buf);
+        ssize_t (*store)(struct bus_type *, const char * buf, size_t count);
 };
 
 Declaring:
